### PR TITLE
Fix TTL: -ttl vs ttl for vault write flags

### PR DESCRIPTION
**Pull request by jpasko1**
Add ability for user to override TTL setting when checking out sts credentials
Add function to force vault login. Useful if about to launch some long running vault things, but still have valid vault creds.

### CODE REVIEW

File: vault.sh [hunk 1]
```diff
@@ -302,31 +335,17 @@ vault_data() {
...
+    cat $HOME/.config/libsh/hc_vaults.json
+}
+
 vault_list() {
     local help=$(cat <<HELP
 ## vault_list
 
 List known vaults from ~/.config/libsh/hc_vaults.json
 
-example:
```
Comment (aaronaddleman) [inline vault.sh:311]: Whats the reason for removing the example?

File: vault.sh [hunk 2]
```diff
@@ -302,31 +335,17 @@ vault_data() {
     cat $HOME/.config/libsh/hc_vaults.json
 }
 
+vault_config() {
```
Comment (aaronaddleman) [inline vault.sh:338]: I like this idea!
Can we add a help block to this as well? Help option allows for us to loop over all functions and build docs in the future.

File: vault.sh [hunk 3]
```diff
@@ -245,9 +245,42 @@ HELP
             *)
                 vault login username=$VAULT_USER
         esac
+    else
+        echo "Already logged into vault"
     fi
 }
 
+vault_login_now() {
```
Comment (aaronaddleman) [inline vault.sh:253]: This is interesting and I can see the use for this!
What if we make this simple and unset the VAULT_TOKEN being tested by vault_login? Then we just call vault_login again to do login?

File: vault.sh [hunk 4]
```diff
@@ -105,13 +105,13 @@ HELP
 
     # clear vars being set
     aws_reset
-
+    TTL=${TTL:-2h}
```
Comment (aaronaddleman) [inline vault.sh:108]: This is a great idea!
What if we make the TTL more unique to vault?
What do you think about renaming to VAULT_TTL?

**Review by aaronaddleman — COMMENTED** [review]
I left some comments for this PR. Would love your feedback.

File: vault.sh (hunk 2 above)
Comment (jpasko1) [reply vault.sh:338]: Like the idea, can work up a help block.

File: vault.sh (hunk 1 above)
Comment (jpasko1) [reply vault.sh:311]: Mistake.

File: vault.sh (hunk 3 above)
Comment (jpasko1) [reply vault.sh:253]: Sure, That sounds good.

File: vault.sh (hunk 4 above)
Comment (jpasko1) [reply vault.sh:108]: VAULT_TTL is more clear than TTL.

**Review by aaronaddleman — APPROVED** [review]

**Pull request merged**